racker/salus-telemetry-resource-management
# Create ResourceManagement label matching query

# Resolves

    SALUS-124 (https://jira.rax.io/browse/SALUS-124)

# What

    Create a query that will match to all of the labels we have provided

# How

    It creates a query function to manually create the SQL query since this is a fairly complex query

## How to test

   I sent API requests to create objects with multiple different labels under different accounts to make sure I limited to the current tenant and matched the query to specific labels.

# Why

    This query incurs fewer JOIN penalties from the suggested SO approach.

# TODO

    Clean up the code. Write unit tests.
